.build/images/dietpi-imager

```diff
@@ -493,7 +502,34 @@
 			G_EXEC partprobe "$FP_SOURCE"
 			G_EXEC partx -u "$FP_SOURCE"
 		fi
-
+		if [[ "$ADD_DOS_PART" == "1" &&  "$PART_TABLE_TYPE" == 'dos' ]]
```

Comment:
It is generally also possible with GPT partitions. All tools used support both. `parted` could be used to be failsafe (as in `dietpi-build`), or `sgdisk` to use a GPT partitioning tool explicitly. But even `sfdisk` should work just fine:
```suggestion
		if [[ $ADD_DOS_PART == 1 ]]
```

Reply:
I know that it is possible with GPT partition tables. It's just that my code so far has only been tested with MBT and  I didn't want to submit untested code.

For your suggested change... shell quoting rules are... well. I'll take your word for it.

Reply:
Within bash-syntax double brackets `[[ ... ]]`, left hand variables never need to be quoted. Right side usually yes, but for integers we skip them. Although consequently right side could be quoted as well, but then single quotes for literal strings.

We can quickly test it with GPT, also e.g. with the GitHub workflows. Lets add support for it with the same PR.

Reply:
TIL (today I learned...)
Will fix the syntax. Will work on the GPT part. I need to check which of the dozen or so boards that I have laying around here from the ADSB Feeder testing uses GPT... Orange Pi 5+ maybe? Yep, good guess. So easy enough to test for me.

What is your deadline for hitting a 'next release'... again... I've been using DietPi for a while, but am new to the development side of it, so I want to make sure I understand your expectations.

Reply:
A release happens every 4 weeks, so the next one will be on September 23rd. The beta is generally released about a week before that (September 16th). Big changes like this are usually merged before the beta release, but it's flexible, and @MichaIng has the final say.